fft/src/KokkosFFT_Transform.hpp

```diff
@@ -469,11 +469,16 @@ void irfft2(const ExecutionSpace& exec_space, const InViewType& in,
 /// \param exec_space [in] Kokkos execution space
 /// \param in [in] Input data (complex)
 /// \param out [out] Ouput data (complex)
-/// \param axes [in] Axes over which FFT is performed (optional)
+/// \param axes [in] Axes over which FFT is performed (default, all axes)
```

Comment:
I think the word `default` is better than `optional`. I mean one cannot only specify the parameter `s` without specifying `axes` and `norm`. The full set `axes`,`norm` and `s` can be interpreted as optional but not really separately ?

Reply:
I forgot a concluding remark, should we consider updating all `(optional)` to `(default, ...)` ?

Reply:
sure